testsuite/documentation/optional.md

```diff
@@ -119,46 +119,48 @@ Inside of the testsuite, the scenarios that are tagged with
 are executed only if the traditional client is available.
 
 
-### Testing with a CentOS minion
+### Testing with a Red Hat-like minion
 
-Using a CentOS minion with the testsuite is not mandatory.
+Using a Red Hat-like minion (CentOS, Alma, Rocky, ...) with the test suite
+is not mandatory.
 
-If you do not want a CentOS minion, do not define `CENTOSMINION` environment
-variable before you run the testsuite. That's all.
+If you do not want a Red Hat-like minion, do not define `RHLIKE_MINION`
+environment variable before you run the test suite. That's all.
```

Comment:
```suggestion
If you do not want a Red Hat-like minion, do not define a `RHLIKE_MINION`
environment variable before you run the test suite. That's all.
```

Reply:
No, this variable is definite and unique, I think it's correct (and better) without the undefinite article.

Reply:
Alright.